### ISSUE TITLE
/visits gives 404

### ISSUE THREAD

**Issue by andreicristianpetcu** [saturnine/redis_analytics#12]
Hi! I installed your gem and it works but it has a link that points to /visits but it gives a 404. /dashboard works fine. What am I doing wrong? here is my code https://github.com/andreicristianpetcu/tribunal/tree/production Thank you!

**Comment by yshahin** [comment]
Same here, any updates in this?

**Comment by deftOfCenter** [comment]
This is a simple solution for the issue.   Looks like the it was just missing the /visits route.  Hope it helps!

**Comment by yshahin** [comment]
add this to the redis_analytics under the initializers file
```
RedisAnalytics::Dashboard::Engine.routes.append do
  get '/visits', to: 'visits#index'
end
```

this should solve the problem temporarily until it gets merged

**Comment by saturnine** [comment]
Closed in #13
